scss/1-settings/_colour-language.scss

```diff
@@ -29,6 +29,7 @@ $cads-language__warning-colour: palette.$cads-palette__red !default;
 $cads-language__success-colour: palette.$cads-palette__green !default;
 
 // Navigation colours
+$cads-language__nav-background-colour: $cads-language__brand-primary !default;
```

Comment:
@marianayovcheva Just an FYI that I've needed to add this and we'll probably need to reconcile things whenever you pick https://github.com/citizensadvice/design-system/pull/1863 back up again